<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="igr_centralwidget">
    <item row="0" column="2">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="3">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="lineEdit_3"/>
    </item>
    <item row="5" column="2">
     <widget class="QFrame" name="fr_Logo">
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>170</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>170</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="lineEdit_4"/>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="lineEdit_5"/>
    </item>
    <item row="3" column="3">
     <widget class="QLineEdit" name="lineEdit_6"/>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="lineEdit_7"/>
    </item>
    <item row="4" column="3">
     <widget class="QLineEdit" name="lineEdit_8"/>
    </item>
    <item row="5" column="1">
     <widget class="QFrame" name="fr_Layouttest">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QFormLayout" name="ifl_Layouttest">
       <item row="0" column="0">
        <widget class="QLineEdit" name="lineEdit_9"/>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_10"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_11"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEdit_12"/>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineEdit_13"/>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="lineEdit_14"/>
       </item>
       <item row="3" column="1">
        <widget class="QDateTimeEdit" name="dateTimeEdit"/>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
